test/unit/error.spec.ts

```diff
@@ -292,3 +285,94 @@ test("return from nested finally", () => {
     `;
     expect(util.transpileAndExecute(code)).toBe("finally AB");
 });
+
+test.each([
+    `"error string"`,
```

Comment:
Might be a bit cleaner to pass actual values there and convert them with util.valueToString

Reply:
True, won't atm due to  `util.valueToString(undefined) = undefined` (#733).